OsmAnd/src/net/osmand/plus/wikipedia/WikipediaPlugin.java

```diff
@@ -161,6 +161,12 @@ public boolean init(@NonNull OsmandApplication app, Activity activity) {
 		return true;
 	}
 
+	@Override
+	public void disable(@NonNull OsmandApplication app) {
+		super.disable(app);
+		toggleWikipediaPoi(false, null);
```

Comment:
We have access to WikiPoiFilter in search even if WikipediaPlugin is disabled because it is stored in PoiFiltersHelper.